bindings/java/src/main/com/apple/foundationdb/NativeFuture.java

```diff
@@ -53,7 +53,7 @@ protected void registerMarshalCallback(Executor executor) {
 		}
 	}
 
-	private void marshalWhenDone() {
+	void marshalWhenDone() {
```

Comment:
Is there somewhere that needs access to this function now?

Reply:
didn't--vestigial change. Thanks for the catch.